### DISCUSSION TITLE
Quests?

### DISCUSSION THREAD

**Discussion by LostLea** [samolego/Taterzens#128] [Ideas]
Could you add something so that we can add Quests???

Pls pls pls

**Comment by samolego** [comment]
Quests as "bring item x"?

**Reply by LostLea** [reply]
yea or smthing like like "Kill 5 sheeps" idk im using ur mod for an adventure map on 1.18.2 and it would be rlly nice
 :)

**Reply by LostLea** [reply]
would that work?

**Reply by samolego** [reply]
It's doable but I don't really work on taterzens that much rn, sorry.

**Reply by LostLea** [reply]
dw :}
